twind.install.js

```diff
@@ -0,0 +1,21 @@
+import { install } from "@twind/core"
```

Comment:
Moving the install code to this external file so it can contain the code to suppress the warning.